src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/StrategiesVmssTests.ps1

```diff
@@ -40,6 +40,149 @@ function Test-SimpleNewVmss
         Assert-AreEqual "2016-Datacenter" $x.VirtualMachineProfile.StorageProfile.ImageReference.Sku
         Assert-NotNull $x.VirtualMachineProfile.NetworkProfile.NetworkInterfaceConfigurations[0].IpConfigurations[0].LoadBalancerBackendAddressPools;
         Assert-NotNull $x.VirtualMachineProfile.NetworkProfile.NetworkInterfaceConfigurations[0].IpConfigurations[0].Subnet
+		Assert-Null $x.Identity  
+    }
+    finally
+    {
+        # Cleanup
+        Clean-ResourceGroup $vmssname
+    }
+}
+
+function Test-SimpleNewVmssWithSystemAssignedIdentity
+{
+    # Setup
+    $vmssname = Get-ResourceName
+
+    try
+    {
+        $username = "admin01"
+        $password = "werWER345#%^" | ConvertTo-SecureString -AsPlainText -Force
```

Comment:
Remove real password

Reply:
@sergey-shandar  what is your take on this?

If I change the password in one place, i'd rather change it in all of them

Reply:
After discussin this with @sergey-shandar, we decided not to change it as of now.

Reason being the vm is destroyed pretty much as soon the status says the vm is ready... 

The vm itself has nothing on it other than a vanilla OS.

Reply:
I'm happy with any solution, if the solution will allow to rerecord the scenario test any time without any additional changes in code or configuring build environment.

Reply:
@hyonholee @sergey-shandar  I opened the [following issue](https://github.com/Azure/azure-powershell/issues/5879) to fix the hardcoded password in the tests.

Reply:
leaving any hardcoded password to public repo is not a good practice.   please replace all password string to [PLACEHOLDER]

Reply:
I'm okay with either replacing the current hardcoded password to the suggested strong (Placeholder) in this PR or OneSDK team providing the solution for generating random password so that this PR can use the functionality to remove the hard coded password.